site/content/3.12/release-notes/version-3.12/incompatible-changes-in-3-12.md

```diff
@@ -170,6 +170,27 @@ onward and will be removed in a future version.
 You can use [Stream Transactions](../../develop/transactions/stream-transactions.md)
 instead in most cases, and in some cases AQL can be sufficient.
 
+## Breaking changes to the `collation` Analyzer
+
+The [`collation` Analyzer](../../index-and-search/analyzers.md#collation) lets
+you adhere to the alphabetic order of a language in range queries. For example,
+using a Swedish locale (`sv`), the sorting order is `å` after `z`, whereas using
```

Comment:
```suggestion
using a Swedish locale (`sv`), the sorting order is `å` after `a`, whereas using
```

Reply:
@MBkkt It's actually abc...xyzåäö, see https://en.wikipedia.org/wiki/Swedish_alphabet

Reply:
ok, but your example isn't clear, you can use same letters for different locales